__tests__/theme/components/LanguageDropdown.spec.js

```diff
@@ -0,0 +1,84 @@
+
+import { shallowMount } from '@vue/test-utils'
+import Header from "@/theme/components/Header.vue"
+
+describe('As user when i change the language', () => {
+  test('if the language is the same stay on the current page', () => {
+    const wrapper = shallowMount(Header, {
+      mocks: {
+        $lang: 'es',
+        $page: {path: '/es'},
+        $site: {
+          themeConfig: {
+            search: true
+          },
+          locales: {
+            "es": {label: 'Español', lang: 'es'}
+          }
+        },
+        $router: {
+          options: {routes: []}
+        }
+      }
+    });
+
+    expect(wrapper.element).toMatchSnapshot()
+  });
+
+  test('if the language changes try to stay on the same page', () => {
```

Comment:
Doesn't look like this test has any `expect` statement? What's its purpose?

Reply:
Is this just a duplicate of the test below? Same title (`if the language changes try to stay on the same page`). Should this test be deleted?

Reply:
no, i just not changed the title test, the behavior of the test is if the language changes and is not in registered at the router set the item path. In this case 'Português' is not at the router so it will change completely path to /por/ and not substitute some part of the path.

Reply:
Shouldn't we still have some expect statement for the test?